scheds/rust/scx_layered/src/main.rs

```diff
@@ -2282,6 +2292,36 @@ impl<'a> Scheduler<'a> {
         Ok(())
     }
 
+    fn refresh_idle_qos(&mut self) -> Result<()> {
+        if !self.idle_qos_enabled {
+            return Ok(());
+        }
+
+        for layer in self.layers.iter() {
+            let idle_resume_us = layer.kind.common().idle_resume_us as i32;
+            for cpu in 0..layer.cpus.len() {
+                if layer.cpus.test_cpu(cpu) {
```

Comment:
Wouldn't this be always true? It's testing whether a cpu from iterating `layer.cpus` would be in`layer.cpus`, no?